docs/docs/api/path-composers/prop.md

```diff
@@ -0,0 +1,33 @@
+---
+id: prop
+title: Prop
+sidebar_label: Prop
+---
+
+`Prop` allow us to refer to [React
```

Comment:
Also, instead of React props, it should be View props.

The operations and modifiers are library neutral and shouldn't contain information about React or other libraries.

Reply:
We should keep information about React only under the `@c11/engine.react` page (which we should make)

Reply:
Just fyi, we already have a page for different engine packages an end user can use. I'd recommend going through the documentation in HTML form instead of directly through code: https://sosinc.io/engine/docs/packages 

Structure in rendered documentation site can differ from code, and should also be easier to explore.